### DISCUSSION TITLE
Renovate doesn't upgrade after minor bump with ignoreUnstable when using prerelease semver versions

### DISCUSSION THREAD

**Discussion by Jolg42** [renovatebot/renovate#8545] [Request Help]
**What Renovate type, platform and version are you using?**

Renovate on GitHub, original repo: https://github.com/prisma/prisma/blob/master/.github/renovate.json

**Describe the bug**

The problem happens with the `@prisma/engines` and `@prisma/engines-version` npm packages which are only released in a semver pre-release style like (from `npm info @prisma/engines versions`)
```
 '2.16.0-1.359e744341f6e278d7f8564b6acfa3fd79531dc8',
  '2.16.0-2.369dae08a1d87eba8ea3ed671381dca74ba0f057',
  '2.16.0-3.4de8f31ec17e56b8766599aa86fee51dbb608c7c',
  [...]
  '2.16.0-43.a973e24a4e5194f8fe3301bfa7ff982b24f97c05',
  '2.16.0-44.854c8ba7f0dce66f115af36af24e66989a8c02a1',
  '2.17.0-1.4ec7e617c76720ebc757144e0bc16b819dd3c038',
  '2.17.0-2.47b8c42bf220a67f979b8d64273a2314a33542e2',
  '2.17.0-3.295a10a0ac32fb6dd59a2e3975ae5dfc568c9372',
  '2.17.0-4.62e1b79e1c93665abe9d3a3694eba727695dcf82'
```

Here is what semver parsing looks like [from my runkit](https://runkit.com/jolg42/5fc0eb476ad334001a972a57)
<img width="753" alt="Screen Shot 2021-02-04 at 14 06 22" src="https://user-images.githubusercontent.com/1328733/106896758-49ba5880-66f2-11eb-8dfa-4b70acb2852f.png">

I created this small repro, where I set the version to `2.16.0-43.a973e24a4e5194f8fe3301bfa7ff982b24f97c05`
Expecting to see renovate upgrading it to the latest version available `2.17.0-4.62e1b79e1c93665abe9d3a3694eba727695dcf82`
Instead it's only suggesting `2.16.0-44.854c8ba7f0dce66f115af36af24e66989a8c02a1` and will never upgrade the minor to 2.17 after that.
https://github.com/Jolg42/repro-renovate-engines-package/pull/1

We thought that `"ignoreUnstable": true,` would change something but it's the same behavior wit or without.

**Relevant debug logs**

```
from https://app.renovatebot.com/dashboard#github/Jolg42/repro-renovate-engines-package/296152739
DEBUG: packageFiles with updates
{
  "config": {
    "npm": [
      {
        "packageFile": "package.json",
        "deps": [
          {
            "depType": "dependencies",
            "depName": "@prisma/engines",
            "currentValue": "2.16.0-43.a973e24a4e5194f8fe3301bfa7ff982b24f97c05",
            "datasource": "npm",
            "prettyDepType": "dependency",
            "lockedVersion": "2.16.0-43.a973e24a4e5194f8fe3301bfa7ff982b24f97c05",
            "depIndex": 0,
            "updates": [
              {
                "fromVersion": "2.16.0-43.a973e24a4e5194f8fe3301bfa7ff982b24f97c05",
                "toVersion": "2.16.0-44.854c8ba7f0dce66f115af36af24e66989a8c02a1",
                "newValue": "2.16.0-44.854c8ba7f0dce66f115af36af24e66989a8c02a1",
                "newMajor": 2,
                "newMinor": 16,
                "updateType": "minor",
                "isSingleVersion": true,
                "releaseTimestamp": "2021-02-02T12:53:11.181Z"
              }
            ],
            "warnings": [],
            "fixedVersion": "2.16.0-43.a973e24a4e5194f8fe3301bfa7ff982b24f97c05"
          },
          {
            "depType": "dependencies",
            "depName": "@prisma/engines-version",
            "currentValue": "2.16.0-43.a973e24a4e5194f8fe3301bfa7ff982b24f97c05",
            "datasource": "npm",
            "prettyDepType": "dependency",
            "lockedVersion": "2.16.0-43.a973e24a4e5194f8fe3301bfa7ff982b24f97c05",
            "depIndex": 1,
            "updates": [
              {
                "fromVersion": "2.16.0-43.a973e24a4e5194f8fe3301bfa7ff982b24f97c05",
                "toVersion": "2.16.0-44.854c8ba7f0dce66f115af36af24e66989a8c02a1",
                "newValue": "2.16.0-44.854c8ba7f0dce66f115af36af24e66989a8c02a1",
                "newMajor": 2,
                "newMinor": 16,
                "updateType": "minor",
                "isSingleVersion": true,
                "releaseTimestamp": "2021-02-02T12:52:55.117Z"
              }
            ],
            "warnings": [],
            "fixedVersion": "2.16.0-43.a973e24a4e5194f8fe3301bfa7ff982b24f97c05"
          }
        ],
        "packageJsonName": "repro-renovate-engines-package",
        "packageFileVersion": "1.0.0",
        "packageJsonType": "library",
        "npmLock": "package-lock.json",
        "skipInstalls": true,
        "constraints": {
          "npm": "<7"
        }
      }
    ]
  }
}
```

**Have you created a minimal reproduction repository?**

https://github.com/Jolg42/repro-renovate-engines-package/
https://github.com/Jolg42/repro-renovate-engines-package/blob/master/renovate.json

- [ ] This is a really small bug, it does not need a reproduction (think small typo)
- [x] I have provided a minimal reproduction repository
- [ ] I don't have time for that, but it happens in a public repository I have linked to
- [ ] I don't have time for that, and cannot share my private repository
- [ ] The nature of this bug means it's impossible to reproduce publicly

**Additional context**

**Comment by rarkins** [comment]
`true` is the default value. you need to set it to `false`

**Reply by Jolg42** [reply]
:palmface: It worked! many thanks @rarkins 💚
